goss.py

```diff
@@ -94,55 +101,44 @@ def main():
         supports_check_mode=False
     )
 
-    test_file_path = module.params['path']  # test file path
+    test_file_path = module.params['path']
     output_format = module.params['format']  # goss output format
     output_file_path = module.params['output_file']
     goss_path = module.params['goss_path']
     vars_path = module.params['vars_path']
 
-    if test_file_path is None:
-        module.fail_json(msg="test file path is null")
-
```

Comment:
Why did you remove this test?

Reply:
Because the module already makes the path mandatory.